vendor/bat-native-ledger/src/bat/ledger/internal/media/twitch.cc

```diff
@@ -508,8 +513,7 @@ void MediaTwitch::OnEmbedResponse(
   std::string author_name;
   braveledger_bat_helper::getJSONValue("author_name", response, &author_name);
 
-  std::string twitchMediaID = visit_data.name;
-  std::string id = (std::string)TWITCH_MEDIA_TYPE + "#author:" + twitchMediaID;
+  std::string id = (std::string)TWITCH_MEDIA_TYPE + "#author:" + user_id;
```

Comment:
please use `GetPublisherKey`. I missed this one when I was refactoring this code

Reply:
Done.